src/renderer/store/mutations.js

```diff
@@ -473,7 +473,18 @@ function orderBookUpdateReceived(state, res) {
 }
 
 function setTradeHistory(state, trades) {
-  state.tradeHistory = trades;
+  if (state.tradeHistory == null ||
+    (trades.marketName !== state.tradeHistory.marketName) ||
+    (trades.trades.length > 0 && state.tradeHistory.trades.length > 0
```

Comment:
Lets optimize this, consider:
```
const newTrades = trades.trades;
const existingTrades = state.tradeHistory.trades;
if (state.tradeHistory == null ||
    (trades.marketName !== state.tradeHistory.marketName) ||
    ((newTrades.length > 0 && existingTrades.length > 0) && 
      (newTrades[0].tradeTime !== existingTrades.tradeTime) ||
      (newTrades[0].quantity !== existingTrades.quantity) ||
      (newTrades[0].price !== existingTrades.price) ||
      (newTrades[0].side !== existingTrades.side))) {
    state.tradeHistory = trades;
```

However, does this really cover all cases that might cause chart reload on websocket reconnect?